src/client/components/sideMenubar/sideMenu.styles.css

```diff
@@ -0,0 +1,38 @@
+@import url('https://fonts.google.com/specimen/Chakra+Petch?preview.text=GuideIT&preview.text_type=custom');
+
+.side_menu_div {
+  width: 100px;
+}
+
+li {
+  background-color: #298f8a;
+  text-align: center;
+  padding-top: 3px;
+  border-right: solid 5px transparent;
+  line-height: 0.8;
+}
+
+.side_menu_whiteborder:hover {
```

Comment:
This is the part that is controlling the indicator bar at the moment, so you need to replace this with logic in the .js file.

Reply:
Hi Beth Thanks for reviewing my PR.. I will change that part.. Thanks again :)